### PR TITLE
Add documentation how to use Spire in own workloads

### PR DESCRIPTION
**Pull request by marcofranssen**

### CODE REVIEW

File: charts/spire/README.md [hunk 1]
```diff
@@ -32,6 +32,43 @@ spec:
...
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: my-app
+ spec:
+   containers:
+     - name: my-app
+       image: "my-app:latest"
+       imagePullPolicy: Always
++      env:
```
Comment (kfox1111) [inline charts/spire/README.md:49]: The ENV bit is hard to describe. It has no effect on any app I know of. So could be confusing to users when it doesn't "work"

**Review by kfox1111 — COMMENTED** [review]
Generally looks good. one comment inline. Not sure what to do with it.

File: charts/spire/README.md (hunk 1 above)
Comment (marcofranssen) [reply charts/spire/README.md:49]: Agree removed. This is specific to an app that uses this env var.
I also didn't use it in this example.

https://github.com/philips-labs/spiffe-vault/blob/main/charts/spiffe-vault/templates/spiffe-vault.yaml

**Review by kfox1111 — APPROVED** [review]
lgtm

**Pull request merged**